src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -1884,13 +1884,16 @@ trait Namers extends MethodSynthesis {
      */
     def annotSig(annotations: List[Tree], annotee: Tree, pred: AnnotationInfo => Boolean): List[AnnotationInfo] =
       annotations filterNot (_ eq null) map { ann =>
-        val ctx = typer.context
```

Comment:
I'm guessing it's better to keep this out of the `lazily`.

Reply:
OK, I realized typer.context is mutable.